### ISSUE TITLE
CollectionView does not scroll horizontal

### ISSUE THREAD

**Issue by DominikErnst** [dotnet/maui#23109]
### Description

I can scroll the content vertical, but not horizontal.

### Steps to Reproduce

1. Create a MAUI App
2. Insert in XAML
`	<Grid RowDefinitions="*,*" ColumnDefinitions="*,*" >
		<CollectionView Grid.Row="1" Grid.Column="0" ItemsSource="{Binding Items}" >
			<CollectionView.ItemTemplate>
				<DataTemplate>
					<Grid ColumnDefinitions="Auto, Auto, Auto, Auto" ColumnSpacing="4" RowSpacing="4">
						<Label Grid.Column="0" Text="{Binding [0]}" />
						<Label Grid.Column="1" Text="{Binding [1]}" />
						<Label Grid.Column="2" Text="{Binding [2]}" />
						<Label Grid.Column="3" Text="{Binding [3]}" />
					</Grid>
				</DataTemplate>
			</CollectionView.ItemTemplate>
		</CollectionView>
	</Grid>`
2.  Insert in 
`    public ObservableCollection<List<string>> Items { get; set; }

    public AboutView()
    {
        InitializeComponent();
        Items = new ObservableCollection<List<string>>();

        for (int i = 0; i < 20; i++)
        {
            List<string> item = new List<string>
                {
                    $"Item {i + 1} - Subitem 1",
                    $"Item {i + 1} - Subitem 2",
                    $"Item {i + 1} - Subitem 3",
                    $"Item {i + 1} - Subitem 4"
                };
            Items.Add(item);
        }

        BindingContext = this;
    }`

### Version with bug

8.0.3 GA

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Did you find any workaround?

NO

**Comment by Marioo1357** [comment]
If you want to scroll Horizontal you should set:

 <CollectionView.ItemsLayout>
       <LinearItemsLayout Orientation="Horizontal" />
 </CollectionView.ItemsLayout>

**Comment by DominikErnst** [comment]
@Marioo1357 yes, but it scrolls **only** horizontal! I need both directions!

**Comment by andrekoehler** [comment]
I have the same issue. With HorizontalScrollBarVisibility="Always" the horizontal scrollbar is calculated and rendered correctly, but it does not react to mouse clicks on Windows.

**Comment by RoiChen001** [comment]
This issue has been verified using Visual Studio 17.12.0 Preview 1.0 (8.0.3 & 8.0.rc.1.9171)& 17.11.0 (7.0.101). Can repro on Windows platform.

**Comment by stefanomasseroli** [comment]
Same issue here...clicking the horizontal scrollbar does make it move it very very slowly actually, but the items inside don't move.

**Comment by DavidM29** [comment]
Same issue, I can add that horizontal scroll does not work on android at all in a vertical collection view. This makes it practically impossible to display data with multiple columns. If needed I can open a new issue for android with a repro.
